dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/LinkageCheckerTest.java

```diff
@@ -658,44 +663,32 @@ public void testGenerateInputClasspathFromLinkageCheckOption_mavenCoordinates_mi
     LinkageCheckerArguments parsedArguments =
         LinkageCheckerArguments.readCommandLine("--artifacts", "com.google.guava:guava-gwt:20.0");
 
-    ImmutableList<Path> inputClasspath = parsedArguments.getInputClasspath();
+    ImmutableList<Path> inputClasspath =
+        classPathBuilder.resolve(parsedArguments.getArtifacts()).getClassPath();
 
     Truth.assertThat(inputClasspath)
         .comparingElementsUsing(PATH_FILE_NAMES)
         .contains("apache-jsp-8.0.9.M3.jar");
   }
 
   @Test
-  public void testGenerateInputClasspathFromLinkageCheckOption_failOnMissingDependency()
-      throws ParseException {
+  public void testGenerateInputClasspathFromLinkageCheckOption_recordMissingDependency()
+      throws ParseException, RepositoryException {
     // tomcat-jasper has missing dependency (not optional):
     //   org.apache.tomcat:tomcat-jasper:jar:8.0.9
     //     org.eclipse.jdt.core.compiler:ecj:jar:4.4RC4 (not found in Maven central)
     LinkageCheckerArguments parsedArguments =
         LinkageCheckerArguments.readCommandLine(
             "--artifacts", "org.apache.tomcat:tomcat-jasper:8.0.9");
 
-    try {
-      parsedArguments.getInputClasspath();
-      Assert.fail(
-          "Because the unavailable dependency is not optional, it should throw an exception");
-    } catch (RepositoryException ex) {
-      Truth.assertThat(ex.getMessage())
-          .startsWith("Unresolved artifacts: org.eclipse.jdt.core.compiler:ecj:jar:4.4RC4");
-    }
-  }
-
-  @Test
-  public void testGenerateInputClasspath_jarFileList()
```

Comment:
This test has been moved to `LinkageCheckerArgumentTest.testGetJarFiles_jarFileList`